tests/function/test_datatypes.py

```diff
@@ -91,22 +91,6 @@ def test_value(self):
             metadata=metadata,
         )
         self.assertEqual(mock_message(), d.value)
-        self.assertEqual(
-            "keys: ['test'], "
-            "value: test_mock_message, "
-            "event_time: 2022-09-12 16:00:00+00:00, "
-            "watermark: 2022-09-12 16:01:00+00:00, "
-            "metadata: id: test_id, num_delivered: 1",
-            str(d),
-        )
-        self.assertEqual(
-            "keys: ['test'], "
-            "value: test_mock_message, "
-            "event_time: 2022-09-12 16:00:00+00:00, "
-            "watermark: 2022-09-12 16:01:00+00:00, "
-            "metadata: id: test_id, num_delivered: 1",
-            repr(d),
-        )
```

Comment:
Why do we remove str() and repr() ?

Reply:
Because dataclasses by themselves implement str and repr methods. if all the other fields are equal, then their str representation will also be equal :)